BotUI/src/app/components/ChatBox/ChatBox.tsx

```diff
@@ -66,8 +70,8 @@ export default function ChatBox({ bot }: ChatBoxProps) {
 
     useEffect(() => {
         if (endChat.state) {
-            const params = handleEndChat();
-            setQueryParams(params);
+            const queryAttributes = handleEndChat();
```

Comment:
Why create this param and not just add it to the state?

Reply:
done